### PR TITLE
HADOOP-16682. ABFS: Removing unnecessary toString() invocations

### PR DESCRIPTION
**Pull request by bilaharith**
Removing unnecessary toString() invokations.

**Driver test results using accounts in Central India**
mvn -T 1C -Dparallel-tests=abfs -Dscale -DtestsThreadCount=8 clean verify

**Account with HNS Support**
[INFO] Tests run: 63, Failures: 0, Errors: 0, Skipped: 0
[WARNING] Tests run: 436, Failures: 0, Errors: 0, Skipped: 74
[WARNING] Tests run: 206, Failures: 0, Errors: 0, Skipped: 24

**Account without HNS support**
[INFO] Tests run: 63, Failures: 0, Errors: 0, Skipped: 0
[WARNING] Tests run: 436, Failures: 0, Errors: 0, Skipped: 248
[WARNING] Tests run: 206, Failures: 0, Errors: 0, Skipped: 24

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 36s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  19m  6s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 36s |  trunk passed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04  |
| +1 :green_heart: |  compile  |   0m 33s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  checkstyle  |   0m 26s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 36s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  14m 53s |  branch has no errors when building and testing our client artifacts.  |
| -1 :x: |  javadoc  |   0m 30s |  hadoop-azure in trunk failed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.  |
| +1 :green_heart: |  javadoc  |   0m 28s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +0 :ok: |  spotbugs  |   0m 55s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 52s |  trunk passed  |
| -0 :warning: |  patch  |   1m 10s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 28s |  the patch passed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04  |
| +1 :green_heart: |  javac  |   0m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 25s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  javac  |   0m 25s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 16s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 27s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  13m 45s |  patch has no errors when building and testing our client artifacts.  |
| -1 :x: |  javadoc  |   0m 24s |  hadoop-azure in the patch failed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.  |
| +1 :green_heart: |  javadoc  |   0m 24s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  findbugs  |   0m 55s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 28s |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 32s |  The patch does not generate ASF License warnings.  |
|  |   |  60m 17s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2144 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 8066992bc738 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 1f71c4ae714 |
| Default Java | Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/1/artifact/out/branch-javadoc-hadoop-tools_hadoop-azure-jdkUbuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/1/artifact/out/patch-javadoc-hadoop-tools_hadoop-azure-jdkUbuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/1/testReport/ |
| Max. process+thread count | 429 (vs. ulimit of 5500) |
| modules | C: hadoop-tools/hadoop-azure U: hadoop-tools/hadoop-azure |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/1/console |
| versions | git=2.17.1 maven=3.6.0 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java [hunk 1]
```diff
@@ -273,8 +273,9 @@ public FSDataOutputStream append(final Path f, final int bufferSize, final Progr
 
   public boolean rename(final Path src, final Path dst) throws IOException {
     LOG.debug(
-        "AzureBlobFileSystem.rename src: {} dst: {}", src.toString(), dst.toString());
+        "AzureBlobFileSystem.rename src: {} dst: {}", src, dst);
```
Comment (DadanielZ) [inline hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java:276]: Merge to line 275

Comment (bilaharith) [reply hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java]: done

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 35s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  19m 44s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 41s |  trunk passed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04  |
| +1 :green_heart: |  compile  |   0m 33s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  checkstyle  |   0m 27s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 37s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  15m  9s |  branch has no errors when building and testing our client artifacts.  |
| -1 :x: |  javadoc  |   0m 30s |  hadoop-azure in trunk failed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.  |
| +1 :green_heart: |  javadoc  |   0m 28s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +0 :ok: |  spotbugs  |   0m 59s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 57s |  trunk passed  |
| -0 :warning: |  patch  |   1m 15s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 31s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 29s |  the patch passed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04  |
| +1 :green_heart: |  javac  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 26s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  javac  |   0m 26s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 16s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 28s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  13m 39s |  patch has no errors when building and testing our client artifacts.  |
| -1 :x: |  javadoc  |   0m 25s |  hadoop-azure in the patch failed with JDK Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.  |
| +1 :green_heart: |  javadoc  |   0m 25s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  findbugs  |   0m 56s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 30s |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 32s |  The patch does not generate ASF License warnings.  |
|  |   |  61m 27s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2144 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux efbec225484f 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 2ba44a73bf2 |
| Default Java | Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/2/artifact/out/branch-javadoc-hadoop-tools_hadoop-azure-jdkUbuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/2/artifact/out/patch-javadoc-hadoop-tools_hadoop-azure-jdkUbuntu-11.0.7+10-post-Ubuntu-2ubuntu218.04.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/2/testReport/ |
| Max. process+thread count | 415 (vs. ulimit of 5500) |
| modules | C: hadoop-tools/hadoop-azure U: hadoop-tools/hadoop-azure |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-2144/2/console |
| versions | git=2.17.1 maven=3.6.0 findbugs=4.0.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**